apps/publisher/models.py

```diff
@@ -43,3 +43,7 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+    @property
+    def user(self):
+        return self.user_user_publisher.first()
```

Comment:
Where are we using this?

Reply:
we are using this in notification to get user from publisher

Reply:
Not sure where that is. Can you share a screenshot or link to the line number?

Reply:
![Screenshot(15)](https://user-images.githubusercontent.com/17536826/152120158-625c73e6-26a2-4953-b2d7-52a16b7994b5.png)